arm-insights/2015-04-01/swagger/insightsClient.json

```diff
@@ -0,0 +1,1123 @@
+        }
+      },
+      "description": "Represents collection of event categories."
+    },
+    "SenderAuthorization": {
+      "properties": {
+        "action": {
+          "type": "string",
+          "description": "Gets or sets the Action."
+        },
+        "condition": {
+          "type": "string",
+          "description": "Gets or sets the condition."
+        },
+        "role": {
+          "type": "string",
+          "description": "Gets or sets the role."
+        },
+        "scope": {
+          "type": "string",
+          "description": "Gets or sets the scope."
+        }
+      },
+      "description": "The authorization class."
+    },
+    "HttpRequestInfo": {
+      "properties": {
+        "clientRequestId": {
+          "type": "string",
+          "description": "Gets or sets the client request id."
+        },
+        "clientIpAddress": {
+          "type": "string",
+          "description": "Gets or sets the client Ip Address"
+        },
+        "method": {
+          "type": "string",
+          "description": "Gets or sets the method."
+        },
+        "uri": {
+          "type": "string",
+          "description": "Gets or sets the Uri."
+        }
+      },
+      "description": "The HttpInfo class."
+    },
+    "EventData": {
+      "required": [ "channels", "level", "eventTimestamp", "submissionTimestamp" ],
+      "properties": {
+        "authorization": {
+          "$ref": "#/definitions/SenderAuthorization",
+          "description": "Gets or sets the authorization.This is the authorization used by the user who has performed the operation that led to this event."
+        },
+        "channels": {
+          "type": "string",
+          "description": "Gets or sets the event channels.The regular event logs, that you see in the Azure Management Portals, flow through the 'Operation' channel.",
+          "enum": [
+            "Admin",
+            "Operation",
+            "Debug",
+            "Analytics"
+          ],
+          "x-ms-enum": {
+            "name": "EventChannels",
+            "modelAsString": false
+          }
+        },
+        "claims": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets the claims"
+        },
+        "caller": {
+          "type": "string",
+          "description": "Gets or sets the caller"
+        },
+        "description": {
+          "type": "string",
+          "description": "Gets or sets the description of the event."
+        },
+        "id": {
+          "type": "string",
+          "description": "Gets or sets the Id"
+        },
+        "eventDataId": {
+          "type": "string",
+          "description": "Gets or sets the event data Id.This is a unique identifier for an event."
+        },
+        "correlationId": {
+          "type": "string",
```

Comment:
This is not necessarily a guid. It is most often a guid, not there is no rule for it to be it.